internal/pkg/cli/app_deploy.go

```diff
@@ -112,10 +118,14 @@ type ecrService interface {
 
 type dockerService interface {
 	Build(uri, tag, path string) error
-	Login(uri string, auth ecr.Auth) error
+	Login(uri, username, password string) error
 	Push(uri, tag string) error
 }
 
+type commandService interface {
```

Comment:
should this interface be public and placed in `internal/pkg/term/command` package?

Reply:
I think we want to let the client define the abstraction instead of dictating the abstraction from the `command` package.